src/schema/v2/me/requestPriceEstimate.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { camelCase } from "lodash"
+import { ResolverContext } from "types/graphql"
+
+const SubmittedPriceEstimateParamsType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "SubmittedPriceEstimateParams",
+  fields: () => ({
+    artworkId: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Artwork ID submitted for estimate",
+    },
+    requesterName: {
+      type: GraphQLString,
+      description: "Name of the requester",
+    },
+    requesterEmail: {
+      type: GraphQLString,
+      description: "Email of the requester",
+    },
+    requesterPhoneNumber: {
+      type: GraphQLString,
+      description: "Phone number of the requester",
+    },
+  }),
+})
+
+const MutationSuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "RequestPriceEstimatedMutationSuccess",
+  isTypeOf: (data) => {
+    return data._type !== "GravityMutationError"
+  },
+  fields: () => ({
+    submittedPriceEstimateParams: {
+      type: SubmittedPriceEstimateParamsType,
+      resolve: (submittedPriceEstimateParams) => {
+        const result = {}
+        const keys = Object.keys(submittedPriceEstimateParams)
+        keys.forEach((key) => {
+          result[camelCase(key)] = submittedPriceEstimateParams[key]
+        })
+        return result
```

Comment:
There is a Lodash implementation for everything :)

Reply:
Unfortunately mapKeys isn't suited for this. But I learnt something. Thank you.

Reply:
Sorry, this doesn't work point-free because `key` is the second argument. This should work:

```suggestion
        return(mapKeys(submittedPriceEstimateParams, (v, k) => camelCase(k)))
```

Reply:
I tried this too. It replaces the key with the camelized original value